backend/proxy/secure_socks_proxy.go

```diff
@@ -39,6 +39,9 @@ var (
 	// PluginSecureSocksProxyServerName is a constant for the GF_SECURE_SOCKS_DATASOURCE_PROXY_SERVER_NAME
 	// environment variable used to specify the server name of the secure socks proxy.
 	PluginSecureSocksProxyServerName = "GF_SECURE_SOCKS_DATASOURCE_PROXY_SERVER_NAME"
+	// PluginSecureSocksProxyAllowInsecure is a constant for the GF_SECURE_SOCKS_DATASOURCE_PROXY_ALLOW_INSECURE
+	// environment variable used to specify if the proxy should use a TLS dialer.
+	PluginSecureSocksProxyAllowInsecure = "GF_SECURE_SOCKS_DATASOURCE_PROXY_ALLOW_INSECURE"
```

Comment:
How is this supposed to be populated? Planning to make further changes to grafana? If so would suggest a PR on grafana now using this branch as sdk dependency

Reply:
Done